src/translation/translation_pt_PT.ts

```diff
@@ -3876,23 +3876,23 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
+        <translation>Ativado durante a gravação e exibe a pasta da gravação atual. Desativado depois da gravação ou quando o gravador não estiver ativado.</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="131"/>
         <source>Recorder status label</source>
-        <translation>Etiqueta do estado do Gravador</translation>
+        <translation>Etiqueta do estado do gravador</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="63"/>
         <source>&lt;b&gt;Directory:&lt;/b&gt; Select &apos;%1&apos; not to register your server with a directory.&lt;br&gt;Or select one of the genres to register with that directory.&lt;br&gt;Or select &apos;%2&apos; and specify a Custom Directory address on the Options tab to register with a custom directory.&lt;br&gt;&lt;br&gt;For any value except &apos;%1&apos;, this server registers with a directory so that a %3 user can select this server in the client connect dialog server list when they choose that directory.&lt;br&gt;&lt;br&gt;The registration of the server is renewed periodically to make sure that all servers in the connect dialog server list are actually available.</source>
         <comment>%1: directory type NONE; %2: directory type CUSTOM; %3 app name, Jamulus</comment>
-        <translation>&lt;b&gt;Diretório:&lt;/b&gt; Seleccione &apos;%1&apos; para não registar o seu servidor com um diretório.&lt;br&gt;Ou selecione um dos géneros para se registar nesse diretório.&lt;br&gt;Ou selecione &apos;%2&apos; e especifique um endereço de Diretório Personalizado no separador Opções para se registar num diretório personalizado.&lt;br&gt;&lt;br&gt;Para qualquer valor exceto &quot;%1&quot;, este servidor regista-se num diretório para que um utilizador %3 possa selecionar este servidor na lista de servidores da caixa de diálogo de ligação do cliente quando escolher esse diretório.&lt;br&gt;&lt;br&gt;O registo do servidor é renovado periodicamente para garantir que todos os servidores da lista de servidores da caixa de diálogo de ligação estão realmente disponíveis.</translation>
+        <translation>&lt;b&gt;Diretoria:&lt;/b&gt; Selecione &apos;%1&apos; para não registar o seu servidor com uma diretoria.&lt;br&gt;Ou selecione um dos géneros para registar com essa diretoria.&lt;br&gt;Ou selecione &apos;%2&apos; e especifique um endereço de &apos;Diretoria Personalizada&apos; no separador das &apos;Opções&apos; para registar com um diretoria personalizada.&lt;br&gt;&lt;br&gt;Para qualquer valor exceto &quot;%1&quot;, este servidor regista com uma diretoria para que um utilizador %3 possa selecionar este servidor na lista de servidores da janela de ligação do cliente quando eles escolhem essa diretoria.&lt;br&gt;&lt;br&gt;O registo do servidor é renovado periodicamente para garantir que todos os servidores da lista de servidores da janela de ligação estão realmente disponíveis.</translation>
```

Comment:
I have doubts about the change here from masculine "Diretório" to feminine "Directoria".

Although I'm not a Portuguese speaker, I have looked at https://en.bab.la/dictionary/portuguese-english/diretoria, and it suggests the masculine form (as originally translated) is more applicable to our usage. "Diretoria" (f) seems to be more a directorate than a directory.